packages/shorebird_cli/lib/src/commands/patch/patch_ios_framework_command.dart

```diff
@@ -0,0 +1,212 @@
+import 'dart:io' hide Platform;
+
+import 'package:crypto/crypto.dart';
+import 'package:mason_logger/mason_logger.dart';
+import 'package:platform/platform.dart';
+import 'package:shorebird_cli/src/code_push_client_wrapper.dart';
+import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/config/shorebird_yaml.dart';
+import 'package:shorebird_cli/src/doctor.dart';
+import 'package:shorebird_cli/src/formatters/file_size_formatter.dart';
+import 'package:shorebird_cli/src/ios.dart';
+import 'package:shorebird_cli/src/logger.dart';
+import 'package:shorebird_cli/src/shorebird_artifact_mixin.dart';
+import 'package:shorebird_cli/src/shorebird_build_mixin.dart';
+import 'package:shorebird_cli/src/shorebird_config_mixin.dart';
+import 'package:shorebird_cli/src/shorebird_environment.dart';
+import 'package:shorebird_cli/src/shorebird_validation_mixin.dart';
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+
+class PatchIosFrameworkCommand extends ShorebirdCommand
+    with
+        ShorebirdConfigMixin,
+        ShorebirdValidationMixin,
+        ShorebirdBuildMixin,
+        ShorebirdArtifactMixin {
+  PatchIosFrameworkCommand({
+    HashFunction? hashFn,
+  }) : _hashFn = hashFn ?? ((m) => sha256.convert(m).toString()) {
+    argParser
+      ..addOption(
```

Comment:
Can we add flavor support now before merging this?

Reply:
Ha, sure! I thought I merged this 😳

Reply:
Done